packages/rest/src/coercion/utils.ts

```diff
@@ -0,0 +1,74 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/rest
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import * as debugModule from 'debug';
+
+const debug = debugModule('loopback:rest:coercion');
+
+export function isEmpty(data: string) {
+  debug('isEmpty %s', data);
+  return data === '';
+}
+/**
+ * A set of truthy values. A data in this set will be coerced to `true`.
+ *
+ * @param data The raw data get from http request
+ * @returns The corresponding coerced boolean type
+ */
+export function isTrue(data: string): boolean {
+  return ['TRUE', '1'].includes(data.toUpperCase());
+}
+
+/**
+ * A set of falsy values. A data in this set will be coerced to `false`.
+ * @param data The raw data get from http request
+ * @returns The corresponding coerced boolean type
+ */
+export function isFalse(data: string): boolean {
+  return ['FALSE', '0'].includes(data.toUpperCase());
+}
+
+export function isSafeInteger(data: number) {
+  return -Number.MAX_SAFE_INTEGER <= data && data <= Number.MAX_SAFE_INTEGER;
```

Comment:
`MIN__SAFE_INTEGER` instead of `-MAX_SAFE_INTEGER`
I think the real MIN_SAFE_INTEGER is -MAX_SAFE_INTEGER - 1 (e.g. 8 bit integer representation can only range from -128 to 127)

Reply:
never mind, apparently this isn't how it works in javascript xD.
You could just use this function here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isSafeInteger

Reply:
cool